rpc_jobs/rpc_aio.yml

```diff
@@ -169,6 +169,7 @@
 
     dsl: |
       // CIT Slave node
+      currentBuild.result = 'SUCCESS'
```

Comment:
Does this mean if a failure happens between here and line 199 the build will be marked as `SUCCESS`?